### PR TITLE
styled the sidebar's element selector

### PR DESCRIPTION
**Pull request by RohitKumarGit**
#### Describe the changes you have made in this PR 
 * added 2 new fonts in sidebar
 * increased the margin of the circuit elements
 * added border and some minor padding changed

### Screenshots of the changes (If any) -
image

### CODE REVIEW

**Comment by RohitKumarGit** [comment]
@nitin10s @Shivansh2407  review this.

File: public/css/sidebar.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.ui-accordion-header {
+    background: red !important;
+    border:none !important;
+    font-family: 'Fira Sans', sans-serif !important;
+    outline: none !important;
+    background: #3d3d46 !important;
+    border-top-right-radius: 3px !important;
+    border-top-left-radius: 10px !important;
+    border-right: 2px solid white !important;
+    border-bottom: 1px solid white !important;
```
Comment (abstrekt) [inline public/css/sidebar.css]: use !important only when necessary.

File: app/views/simulator/edit.html.erb [hunk 2]
```diff
@@ -4,6 +4,8 @@
 <!-- <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css"> -->
 <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384&#45;BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"> -->
 <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+<link href="https://fonts.googleapis.com/css?family=Fira+Sans|Roboto+Mono&display=swap" rel="stylesheet">
```
Comment (abstrekt) [inline app/views/simulator/edit.html.erb]: importing new fonts increased website load, not good if the font is just used here & not widely.

File: app/views/simulator/edit.html.erb [hunk 3]
```diff
@@ -4,6 +4,8 @@
...
+<link rel="stylesheet" type="text/css" href="/css/sidebar.css">
```
Comment (abstrekt) [inline app/views/simulator/edit.html.erb]: try not to create new stylesheets, add styles to dedicated stylesheets.

File: public/css/sidebar.css (hunk 1 above)
Comment (RohitKumarGit) [reply public/css/sidebar.css]: Yea , I can use specificity later on in a commit , was just playing with UI

File: app/views/simulator/edit.html.erb (hunk 2 above)
Comment (RohitKumarGit) [reply app/views/simulator/edit.html.erb]: It's the most fast font available for load , then u should also consider removing the bootstrap components which you are not using?

File: app/views/simulator/edit.html.erb (hunk 3 above)
Comment (RohitKumarGit) [reply app/views/simulator/edit.html.erb]: yea , okay fine

**Comment by RohitKumarGit** [comment]
> We've already increased the margins in another PR, what's the purpose of adding two fonts? The PR is fine but the thing is (since we have this as a project), we are wanting to work on this holistically, and not just adding little margins and changing fonts. So, let's put this PR on hold for now.

Sir I have also changed colors, It's not just two fonts

File: app/views/simulator/edit.html.erb (hunk 2 above)
Comment (abstrekt) [reply app/views/simulator/edit.html.erb]: 🤦‍♂

**Comment by nitin10s** [comment]
Closing this since we are revamping the simulator completely.

**Pull request closed without merging**